deltachat-ffi/deltachat.h

```diff
@@ -1655,8 +1679,9 @@ char*           dc_get_mime_headers          (dc_context_t* context, uint32_t ms
  */
 void            dc_delete_msgs               (dc_context_t* context, const uint32_t* msg_ids, int msg_cnt);
 
-/**
+/*
```

Comment:
This is not a doxygen comment anymore, any reason for that?

Reply:
yip, i want to exclude dc_empty_server() from the doc.